crates/storage/store/engines/libmdbx.rs

```diff
@@ -580,6 +609,45 @@ impl Encodable for ChainDataIndex {
     }
 }
 
+impl redb::Value for ChainDataIndex {
```

Comment:
We shouldn't have code necessary for the redb impl in a module gated behind the `libmdbx` feature, maybe we could move `ChainDataIndex` outside this module ( to engine or engine/utils) and only keep implementation-specific code here

Reply:
Agreed, I moved the `ChainDataIndex` enum to its own `utils` module under `engines` here [dbf7bdd](https://github.com/lambdaclass/ethrex/pull/1351/commits/dbf7bdd9375f1f2f2b8284f859eddb55a2ed6c74)